jep/200/README.adoc

```diff
@@ -209,6 +212,7 @@ There is an obvious risk that some plugins will have a legitimate need to serial
 In fact it is expected that there will be some such cases;
 this is simply the cost of having a tighter security policy.
 
+// oleg-nenashev: TODO: Clarify, which plugins have been tested and which plugins in the setup wizard were not covered by tests
```

Comment:
Those plugins bundled in CloudBees products have been tested.

Reply:
It does not say anything to a common Jenkins contributor. The list needs to be provided. The main idea is to understand which community-recommended plugins in the Setup Wizard are not covered by tests

Reply:
What was covered...

1) Everything in ATH
2) PCT for a set of plugins, @jglick and @oleg-nenashev will construct a list. Generally, "compatible" and "verified" plugins from https://go.cloudbees.com/plugins/
3) @oleg-nenashev will review the uncovered plugins, and then we will plan testing for them before the JEP changes get released